### ISSUE TITLE
Duplicate bind9 ACL for a single IP

### ISSUE THREAD

**Issue by bsaglietto** [keexybox/keexyapp#11]
Duplicate ACL for a single IP when Bind9 configuration is generated. This creates behavior where the wrong profile is used for DNS filtering.

Reported on our forum: https://forum.keexybox.org/Thread-Weird-behaviour-on-Profiles-Devices

**Comment by bsaglietto** [comment]
The bug is related to the list of IP not reset after the generation of ACLs. The value of $params['ips'] is not reset on each profile iteration, so each profile's IP addresses accumulate for each subsequent acl_profile that is regenerated.

**Comment by bsaglietto** [comment]
You can fix it on your KeexyBox by following the instructions below :

- Connect over SSH to your raspberryPI
- As root run :

```
su - keexybox
cd /opt/keexybox/keexyapp/src/Controller/
mv ConfigController.php ConfigController.php.bak
wget https://raw.githubusercontent.com/keexybox/keexyapp/master/src/Shell/ConfigShell.php
```

**Comment by bsaglietto** [comment]
Fixed from release 20.10.2
